src/drivers/intel/cavs/hda-dma.c

```diff
@@ -122,6 +122,15 @@
 #define HDA_DMA_PTR_DBG_NUM_CP	32 /* number of traces to output after start */
 
 #if HDA_DMA_PTR_DBG
+
+enum hda_dbg_src {
+	HDA_DBG_HOST = 0,
```

Comment:
Can you comment what each enum value means wrt the DMA transfer. I'm good after this so @mmaka1 can merge if he also approves.

Reply:
updated